src/app/course/event/event-stats-bar-chart/event-stats-bar-chart.component.ts

```diff
@@ -15,14 +16,17 @@ export class EventStatsBarChartComponent implements OnInit {
 
     ngOnInit(): void {
         this.labelsX = Object.keys(this.answers).map(convert)
+        for (let i = 0; i < this.labelsX.length; i++) {
+            if (this.labelsX[i].length > 90) {
+                this.labelsX[i] = this.labelsX[i].slice(0, 90) + '...'
+            }
+        }
```

Comment:
```suggestion
        this.labelsX = Object.keys(this.answers)
        .map(convert)
        .map(label => label.length < 90 ? label : lable.slice(0, 90) + '...')
```

Very good solution, thank you! But here is a slightly better implementation of it.

Reply:
I wrote this on github, make sure it works properly and format the file if needed.